bundles/org.openhab.binding.samsungtv/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file as since openHAB 4.0 it is replaced by the addon.xml

Reply:
Removed